tests/conftest.py

```diff
@@ -39,7 +70,7 @@ def SOLOVEV(tmpdir_factory):
     print("exec_dir=", exec_dir)
     print("cwd=", cwd)
 
-    args = ["-o", str(desc_h5_path), input_filename, "--numpy", "-vv"]
+    args = ["-o", str(desc_h5_path), input_filename, "-vv"]
```

Comment:
I'd like to keep the ``--numpy`` flag for now, so that it actually tests that backend (until we stop supporting it formally). You could maybe make a separate one using the JAX backend just for benchmarking?